src/Components/UI/Forms/ProductForm/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useCallback } from "react";
+import { useFormik, FormikValues } from "formik";
+import { IProduct } from "../../../../models";
+import { InputText } from "primereact/inputtext";
+import styled from "styled-components";
+import { Button } from "primereact/button";
+import uuid from "react-uuid";
+import { QueryClient, useMutation } from "react-query";
+import { apiUrl, URL } from "../../../../shared";
+import { ProductService } from "../../../../shared/services/product-service";
+import { Link } from "react-router-dom";
+
+const Root = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 30px;
+`;
+
+const createProduct = (product: Partial<IProduct>) =>
+  ProductService.addProduct(product as IProduct);
+
+export const ProductForm = () => {
+  const { mutate } = useMutation(createProduct, {});
+
+  const handleFormSubmit = useCallback((data: Partial<IProduct>) => {
+    console.log(data, "dsd");
+    mutate(data);
+  }, []);
+
+  const formik = useFormik({
+    initialValues: {
+      name: " ",
+      brand: " ",
+      image: " ",
+    },
+
+    onSubmit: handleFormSubmit,
+  });
+  return (
+    <Root>
+      <form onSubmit={formik.handleSubmit}>
+        <h1>Form Product</h1>
+        <span className="p-float-label mt-4">
+          <InputText
+            id="price"
+            name="price"
+            type="text"
+            onChange={formik.handleChange}
+            value={formik.values.price || " "}
+          />
```

Comment:
Which fields are required? I think all..

```suggestion
          <InputText
            id="price"
            name="price"
            type="text"
            required
            onChange={formik.handleChange}
            value={formik.values.price || " "}
          />
```

Reply:
Fixed